chaco/examples/demo/advanced/scalar_image_function_inspector.py

```diff
@@ -621,12 +621,8 @@ def configure_traits(self, *args, **kws):
         return super(ModelView, self).configure_traits(*args, **kws)
 
 
-def show_plot(**kwargs):
-    model = Model(**kwargs)
-    view = PlotUI(**kwargs)
-    modelview = ModelView(model=model, view=view)
-    modelview.configure_traits()
+demo = ModelView(model=Model(), view=PlotUI())
 
 
 if __name__ == "__main__":
-    show_plot()
+    demo.configure_traits()
```

Comment:
This would look / work a lot better as a popup, but that leads to very strange errors due to etsdemo internals.  Doing so I get the following error:
```
Traceback (most recent call last):
  File "/Users/aayres/.edm/envs/test-chaco/lib/python3.6/site-packages/traitsui/qt4/instance_editor.py", line 428, in edit_instance
    view, kind=factory.kind, id=factory.id
  File "<string>", line 603, in edit_traits
  File "<string>", line 599, in _start_timer
  File "/Users/aayres/.edm/envs/test-chaco/lib/python3.6/site-packages/traits/trait_types.py", line 3360, in validate
    self.resolve_class(object, name, value)
  File "/Users/aayres/.edm/envs/test-chaco/lib/python3.6/site-packages/traits/trait_types.py", line 3462, in resolve_class
    super().resolve_class(object, name, value)
  File "/Users/aayres/.edm/envs/test-chaco/lib/python3.6/site-packages/traits/trait_types.py", line 3178, in resolve_class
    self.validate_failed(object, name, value)
  File "/Users/aayres/.edm/envs/test-chaco/lib/python3.6/site-packages/traits/trait_types.py", line 3207, in validate_failed
    self.error(object, name, value)
  File "/Users/aayres/.edm/envs/test-chaco/lib/python3.6/site-packages/traits/base_trait_handler.py", line 75, in error
    object, name, self.full_info(object, name, value), value
traits.trait_errors.TraitError: The 'model_view' trait of a TimerController instance must be a ModelView or None, but a value of <___main___.ModelView object at 0x7fa7c4a40150> <class '___main___.ModelView'> was specified.
Abort trap: 6
```

we have a trait defined as `a = Instance("B")` where `B` is a class defined elsewhere in the module.  However, due to etsdemo messing with locals to change `__name__` to `___name___`, we end up passing an instance of `___name___.B` which traits catches as an invalid value...
I do not know how to resolve this. Perhaps we just leave it as demo embedded in the etsdemo ui for now (it is resizable anyway so if you stretch the etsdemo ui out enough you can see the full demo).

@rahulporuri any ideas on how we could resolve this?  It is awkward because you need to do `Instance("ModelView")` not `Instance(ModelView)` because `ModelView` is defined after `TimerController` in the file, _and_ `ModelView` also has a trait which is `Instance(TimerController)` so we can't simply move the class to be defined above.

Reply:
0. if possible, can we rename it to something other than `ModelView` because that slightly confuses things
1. I don't think it's bad to use `Instance("ModelView")` instead of `Instance(ModelView)`. I think there are definitely valid usecases (like here) where a class is defined later in the module which needs to be used here.

Reply:
Note:
 switching to 
 ```
 model_view = Instance(
        "chaco.examples.demo.advanced.scalar_image_function_inspector.DemoModelView"  # noqa: E501
    )
```
did not solve the issue.  I will leave it as a demo not a pop up for now